server/src/main/java/org/elasticsearch/action/bulk/TransportShardBulkAction.java

```diff
@@ -487,9 +487,9 @@ private static void onComplete(
         final BulkItemResponse executionResult = context.getExecutionResult();
         final boolean isFailed = executionResult.isFailed();
         if (isFailed == false && opType != DocWriteRequest.OpType.DELETE) {
-            DocumentSizeReporter documentSizeReporter = documentParsingProvider.getDocumentParsingReporter(docWriteRequest.index());
+            DocumentSizeReporter documentSizeReporter = documentParsingProvider.getDocumentParsingReporter(docWriteRequest.index(), false);
```

Comment:
Is it right to hardcode false here?

Reply:
this place is only going to be used for RA-Ingest metric - hence it does not matter if the index is timeseries or not.
However it is confusing when reading this code now. I think we should remove this in favour of having this in serverless' IndexEngine#index